platform/ui/index.js

```diff
@@ -9,4 +11,6 @@ export {
   Typography,
 } from './src/components';
 
+export { utils };
```

Comment:
This is okay for now, but we should have a better name than `utils`. That namespace quickly becomes a catch-all bucket.